src/backend/joanie/debug/views.py

```diff
@@ -312,7 +312,7 @@ def get_context_data(self, **kwargs):
         product.save()
         order = OrderFactory(owner=owner, product=product)
         billing_address = BillingAddressDictFactory()
-        credit_card = CreditCard.objects.filter(owner=owner).first()
+        credit_card = CreditCard.objects.get(owner=owner, is_main=True)
```

Comment:
My bad, I realize that if the owner has no card, it will 😅 
```suggestion
        credit_card = CreditCard.objects.filter(owner=owner, is_main=True).first()
```